src/pilot/deploy-overcloud.py

```diff
@@ -430,19 +815,20 @@ def main():
         start = time.time()
         status = run_deploy_command(cmd)
         end = time.time()
-        print '\nExecution time: {} (hh:mm:ss)'.format(
-            time.strftime('%H:%M:%S', time.gmtime(end - start)))
+        logger.info('\nExecution time: {} (hh:mm:ss)'.format(
+            time.strftime('%H:%M:%S', time.gmtime(end - start))))
         print 'Fetching SSH keys...'
         update_ssh_config()
         if status == 0:
             horizon_url = finalize_overcloud()
+            logger.info("\nDeployment Completed")
         else:
             horizon_url = None
-        print 'Overcloud nodes:'
+        logger.info('Overcloud nodes:')
         identify_nodes()
 
         if horizon_url:
-            print '\nHorizon Dashboard URL: {}\n'.format(horizon_url)
+            logger.info('\nHorizon Dashboard URL: {}\n'.format(horizon_url))
     except ValueError as err:
```

Comment:
The above line should be changed to:
except Exception as err:
so it picks up failure in the new create_* code and failure of compute nodes to reboot.

Reply:
Agree. Changed.